### ISSUE TITLE
[java] enum for int array initialized with strings

### ISSUE THREAD

**Issue by etherealjoy** [OpenAPITools/openapi-generator#49]
##### Description

The generated code for the enums of int types, in an array of said enums is initialized with a string.
Compilation fails
```java
  public enum TypeEnum {
    _0("0"),
    
    _1("1"),
    
    _2("2"),
    
    _3("3")
```
##### openapi-generator version

master

##### OpenAPI declaration file content or url

```yaml
  TypeIds:
    type: object
    properties:
      type:
        type: array
        items:
          type: integer
          format: int32
          enum:
          - 0
          - 1
          - 2
          - 3
```
##### Command line used for generation

java -jar ./openapi-generator-cli.jar generate  -i ./Api.yaml  -o ./genjava -l java

**Comment by jmini** [comment]
This is a nice input. We should support this. Thank you for reporting it.

**Comment by jmini** [comment]
This is fixed for java and kotlin with the latest `3.1.0-SNAPSHOT` version. Can you verify?

**Comment by etherealjoy** [comment]
I will test it and will close this issue if solved.

**Comment by etherealjoy** [comment]
Solved with #75 
Generated code looks like this now,
```java
    NUMBER_0(0),
    
    NUMBER_1(1),
    
    NUMBER_2(2),
    
    NUMBER_3(3);
```
